### ISSUE TITLE
Skip lines when parsing a CSV

### ISSUE THREAD

**Issue by TomBor** [uwdata/arquero#165]
I regularly found and use CSV that had metadata before header and values.
One example here with NOAA data of CO2 concentration in Mauna Loa station : https://www.esrl.noaa.gov/gmd/ccgg/trends/data.html
Direct access to one of these CSV : https://www.esrl.noaa.gov/gmd/webdata/ccgg/trends/co2/co2_annmean_mlo.csv

It would be great to have a new option to `aq.fromCSV` that would skip a desired number of line.

**Comment by jheer** [comment]
Thanks for the recommendation. As you suggest, a simple addition could be to add a parameter that indicates a number of initial lines to skip when reading in the file. An alternative could be to instead indicate a comment / metadata pattern (e.g., `'#'`), such that all lines at the beginning of the file that start with that pattern are skipped until a non-matching line is found.

Any thoughts or opinions on these or other approaches?

**Comment by jheer** [comment]
I'm looking at R's [readr](https://readr.tidyverse.org/) package for inspiration. Of particular interest is: "You can use skip = n to skip the first n lines; or use comment = "#" to drop all lines that start with (e.g.) #."

**Comment by ericemc3** [comment]
From my experience with csv files, skip = n probably represents the most common case, but the possibility to deal with both scenarios would be great

**Comment by TomBor** [comment]
Agreed with @ericemc3 , comment = is a more specific case than skip =.
Another exotic use of CSV from UNHCR portal data who don't use a comment character for metadata:
```
 Extracted:,"United Nations High Commissioner for Refugees"
"Date extracted:","11/05/2021 07:19 UTC"
"Last updated:","08 December 2020"
"Some data presented may differ from statistics published previously due to retroactive changes or the inclusion of previously unavailable data."
"Data table contains the following:","Population figures"
"Content type","End year stock population totals"
"Data quick link",https://www.unhcr.org/refugee-statistics/download/?url=Ovww0M
"Data definitions",https://www.unhcr.org/refugee-statistics/methodology/
"Usage licence",https://creativecommons.org/licenses/by/4.0/
"Population types included","REF,ROC,ASY,IDP,IOC,VDA,OOC,STA"
Years:,"2020,2020"
Origin:,all
"Country of asylum:",all
Year,"Country of origin","Country of origin (ISO)","Country of asylum","Country of asylum (ISO)","Refugees under UNHCR's mandate",Asylum-seekers,"IDPs of concern to UNHCR","Venezuelans displaced abroad"
2020,Afghanistan,AFG,Afghanistan,AFG,0,0,2647358,
2020,Bangladesh,BGD,Afghanistan,AFG,0,5,0,
2020,"Iran (Islamic Rep. of)",IRN,Afghanistan,AFG,39,27,0,
```

**Comment by jheer** [comment]
The `skip` and `comment` options for `fromCSV` are now included in [v4.6.0](https://github.com/uwdata/arquero/releases/tag/v4.6.0).

**Comment by TomBor** [comment]
Great news!
